styles/responsiveipad.css

```diff
@@ -0,0 +1,249 @@
+@media screen and (min-width:768px) {
+    
+    .menu_checkbox__style{
+        display: none;   
+    }
+
+    .menu__style{       
+       position: relative;
+       left: -200px;       
+    }
+    
+    .header_nav__style{        
+        position: relative;
+        top: 0px;
+        left: 0;       
+        width: 65%;
+        height: 30px;
+        z-index: 99;              
+    }
+
+    .header_nav__style-ul{
+        display: flex; 
+        flex-direction: row;
+        margin: 0;
+        padding: 0;
+        position: relative;
+        top: -10px;
+        left: 20px;
+        z-index: 98;
+    }
+
+    .fresh_life_content__style{
+        position: relative;
+        left: 30px;        
+    }
+
+    .about_us_container__style{
+        background-color: #00c4c4;
+        width: 100%;
+        height: 445px;
+        position: relative; 
+        z-index: 1; 
+    }
+    
+    .about_us_content__style{
+        background-position: -20% 40%;        
+        position: relative; 
+        top: -29px; 
+    }
+
+    .about_us_content-h2__style{        
```

Comment:
All of the selectors for this class have `position: relative` rule in common. You can add extra class and keep there styles that are shared